### ISSUE TITLE
Add an option to save even if "This TiddlyWiki is already open in another tab"

### ISSUE THREAD

**Issue by YakovL** [TiddlyWiki/TiddlyFox#51]
Current version strictly prohibit saving of a TW when that file is already open in another tab. However, in my workflow I sometimes open 2-3 copies of the same TW and need to open a new copy to edit while 1-2 copies are open to see some data. Due to the limited number of tabs accessible by ctrl+tab and another issues it is incovenient to start using an already open copy of TW to edit and open a new one to open all the tiddlers that were open in the first one.
Long story short, sometimes I need to save a TW already open in another tab anyway. Either an option to allow this (but still show the warning) or an "are you sure?" dialogue while saving would be very helpful.
Best regards,
Yakov.
